src/App.js

```diff
@@ -44,34 +46,39 @@ function App() {
     setCollapsed(!collapsed);
   };
 
-  const handleNav = (path, label) => {
+  const getSubmenuKey = (location) => {
+    let arr = location.split('/');
+    return arr[1];
```

Comment:
why don't you go ahead and 1 line these, return location.split('/')[1]. 
also, what id location is null?

Reply:
dont think it can be null